packages/concerto-ui-react/package.json

```diff
@@ -20,13 +22,15 @@
   },
   "peerDependencies": {
     "react": "^16.11.0",
-    "react-dom": "^16.11.0"
+    "react-dom": "^16.11.0",
+    "semantic-ui-css": "^2.4.1",
+    "semantic-ui-react": "^0.87.2"
   },
   "devDependencies": {
     "@babel/cli": "^7.7.0",
     "@babel/core": "^7.7.0",
     "@babel/plugin-transform-runtime": "^7.8.3",
-    "@babel/preset-env": "^7.7.1",
+    "@babel/preset-env": "^7.8.4",
```

Comment:
I had attempted to use the same dependencies as in `cicero-ui` thinking that consistency across multiple projects would help maintenance, avoid obscure bugs, etc. Generally speaking I never know why version X or Y in that wall of dependencies and I'm not sure if we have guidelines around that.

https://github.com/accordproject/cicero-ui/blob/a48c80c2e3da7dae6baf56efb3851605f3bc3cf7/package.json#L48

It would be great if the React/Webpack/Babel etc. experts in the room would weigh in on this.